endpoints/endpoints.kaltura.php

```diff
@@ -291,19 +312,6 @@ public static function getKalturaUploadStatus($request)
         ];
     }
 
-    /**
-     * Save Kaltura upload status
-     */
-    public static function saveVideoUploadStatus($video, $videosToUpload, $newStatus, $kalturaData = null)
-    {
-        $video->uploadStatus = $newStatus;
-        self::_updateUploadLog($videosToUpload);
-
-        $node = new TapestryNode($video->tapestryID, $video->nodeID);
-        self::_saveVideoUploadStatusInNode($node, $newStatus, $kalturaData);
-        return $node;
-    }
-
```

Comment:
@legendword can you please confirm that this was removed intentionally? Sorry I merged this in, but then noticed this right after.